torture/src/supervisor/controller.rs

```diff
@@ -121,10 +124,13 @@ impl SpawnedAgentController {
 ///
 /// The controller is placed in the `place` argument. `place` must be `None` when calling this
 /// function.
-pub async fn spawn_agent_into(place: &mut Option<SpawnedAgentController>) -> Result<()> {
+pub async fn spawn_agent_into(
+    place: &mut Option<SpawnedAgentController>,
+    output_path: PathBuf,
```

Comment:
That's a bit of a lie, because it's not really a path to a file where the output will be collected but rather this is literally the workload dir. Can be fixed as a follow up.